frontend_app_workspace/src/container/PopupCreateWorkspace.jsx

```diff
@@ -192,7 +192,11 @@ export class PopupCreateWorkspace extends React.Component {
     switch (fetchGetAllowedSpaceTypes.apiResponse.status) {
       case 200: {
         const apiTypeList = fetchGetAllowedSpaceTypes.body.items
-        this.setState({ allowedTypes: SPACE_TYPE_LIST.filter(type => apiTypeList.some(apiType => apiType === type.slug)) })
+        const allowedTypesList = SPACE_TYPE_LIST.reverse().filter(type => apiTypeList.some(apiType => apiType === type.slug))
+        this.setState({
+          allowedTypes: allowedTypesList,
+          newType: allowedTypesList[0].slug
```

Comment:
This is a bit fragile as it requires the `SPACE_TYPE_LIST` to be in the right order to work.
Wouldn't searching for `CONFIDENTIAL` in the list (and fallback to the first item if `CONFIDENTIAL` is not found) a more robust approach?

Reply:
I think it would be the right approach but I'm fine with introducing some technical tebt for now if it's too time consuming today

Reply:
Updated